### ISSUE TITLE
Remove deprecated collections from size count

### ISSUE THREAD

**Issue by SibylSchaefer** [Chronopolis-Digital-Preservation/audit-control-environment#64]
The data size count for the UCSD-Lib deposits has an incorrect TB size on the TDL ACE:

[+]ucsd-lib | 808233 | 229.4 TB -- | -- | --

TDL removes UCSD data prior to ingesting new data, so this number should only represent the data actually in the system. UCSD does not have 229 TB in TDL, so it looks like it is counting the removed collections as well. This makes it difficult to determine how many TB are actually there without having to manually add up all of the collections that have not been removed.

**Comment by lsitu** [comment]
@SibylSchaefer Is the active collection size 808233 correct on `TDL ACE`? Do we only this issue on `TDL ACE`, or it appears in other nodes (like UCSD ACE), as well?

**Comment by lsitu** [comment]
Also, we have the "PRESERVATION STORAGE" option in the search form, which will list collections by group with the stats for active collections only (collections that were REMOVED are excluded). Could we try that option to see whether it looks correct or not?

**Comment by SibylSchaefer** [comment]
I don't know what the current file size of the UCSD collection is. I just know that if the TB listed only are ones in preservation storage, the number should be closer to 100TB, not 229. I can't use the other ACE instances to compare without subtracting each individual collection from the previous deposit since we don't remove those collections until after replication across nodes is complete. I also don't see any option for searching "Preservation Storage" in the search form.

**Comment by lsitu** [comment]
Thanks @SibylSchaefer for the clariication.
@thorucsd @rstanonik Do you have any ideas regarding the status of ACE on TDL? We have the the "PRESERVATION STORAGE" option on the search form, but it look like it's missing from the TDL version. Also, Is the search results with the "PRESERVATION STORAGE" option correct on UCSD ACE or not?

**Comment by rstanonik** [comment]
TDL is still auditing the latest UCSD collections.  Their ACE size won't be correct until that finishes, right?

**Comment by thorucsd** [comment]
I also don't know that they have actually deleted the old copy there?  When I asked if they had deleted at the beginning of ingest, Courtney seemed to indicate that we didn't need to wait.

**Comment by lsitu** [comment]
@thorucsd @rstanonik Do they deployed the latest version of ace-am? With the latest ace-am, if they'd deleted the old collections, I think the size should be showing up correctly while searching with "PRESERVATION STORAGE" option on the status dropdown, at lease the disk size is smaller if auditing is not done yet. But the number of the collection size should always be correct since those deleted collections are excluded from counting for "PRESERVATION STORAGE".

> TDL is still auditing the latest UCSD collections. Their ACE size won't be correct until that finishes, right?

Yes. I think the size should be smaller in this case since we are counting for the MonitorItem's that are auditing/audited. But if they haven't deleted the old collections, then it'll be larger.

**Comment by thorucsd** [comment]
They deployed the latest version.

**Comment by lsitu** [comment]
Thanks for confirming it @thorucsd .
Can we check whether they've deleted the old collections and make sure they can get the right size while searching with Status="PRESERVATION STORAGE"?

**Comment by thorucsd** [comment]
It might be good to wait until they finish the ACE audit on the latest UCSD submission?

**Comment by lsitu** [comment]
Sounds good. Thanks.

**Comment by SibylSchaefer** [comment]
Circling back to this: the UCSD and TDL ACE interfaces are both reporting the same number of files(869399) and size (869399). However, you can see that TDL has removed the 2022 deposit and UCSD has not. Attaching two screenshots, the first is the UCSD ACE. 
![UCSD_ACe](https://github.com/Chronopolis-Digital-Preservation/audit-control-environment/assets/372729/4d04f542-7fb9-434f-a72f-073f1eb9d93c)
![TDL_ACE](https://github.com/Chronopolis-Digital-Preservation/audit-control-environment/assets/372729/f9e38e2a-05df-4c5b-b545-0bca4a261623)

**Comment by lsitu** [comment]
@thorucsd @rstanonik Could we deploy the latest ACE to chrondev? It looks like our latest PR https://github.com/Chronopolis-Digital-Preservation/audit-control-environment/pull/62 with "PRESERVATION STORAGE"  State search option is missing. Thanks.

**Comment by rstanonik** [comment]
@lsitu We git clone from github, built and deployed ACE to chrondev.
Do you want to test that, then we deploy to chron?
Thanks.
Ron

**Comment by lsitu** [comment]
@rstanonik Thanks. I think it looks good on chrondev with the PRESERVATION_STORAGE status option showing up as expected:
<img width="1440" alt="Screen Shot - chron ACE with preservation storage option " src="https://github.com/Chronopolis-Digital-Preservation/audit-control-environment/assets/2430784/7ccd5827-37f7-4a45-9c9a-36334e0e677e">

It looks like it's good to make it by default. And I've added PR https://github.com/Chronopolis-Digital-Preservation/audit-control-environment/pull/65 so it will always show the active collections when Status page loaded. 
@SibylSchaefer Do we want to do this way so when we click on the `Status` menu on the top, it'll always show the active collections with REMOVED collections excluded from size count?

**Comment by SibylSchaefer** [comment]
@lsitu Is there any way that only the numbers for the active collections are represented in the ACE totals overall, not just when you click Status? It's not intuitive that the home page shows both removed and preserved collections in the total amount.

**Comment by lsitu** [comment]
@SibylSchaefer I think it should always show the active collections once we deploy PR https://github.com/Chronopolis-Digital-Preservation/audit-control-environment/pull/65, the only exception that we can see all collections that were REMOVE is leave the State field blank and Search for it. 
I think we can change it to always show the active collections unless we search it with the REMOVED option from the State field. That is, those REMOVED collections won't show up unless we search for them. Does it sounds better?

**Comment by SibylSchaefer** [comment]
@lsitu Yes, having REMOVED collections only show up when searched for does sound better, thanks!

**Comment by lsitu** [comment]
@SibylSchaefer It sounds good and I've opened PR https://github.com/Chronopolis-Digital-Preservation/audit-control-environment/pull/66 for it. 

Note: Once PR#66 above is deployed, we'll only see REMOVED collections when we search it with the `REMOVED` option selected from the bottom of the `State` dropdown field on the search form. And if we leave the `State` dropdown blank with no state selected, then it'll always search from the active collections with REMOVED collections excluded.

**Comment by rstanonik** [comment]
@lsitu When @thorucsd and I compiled ACE, the pom.xml required lib-artifacts

        <repository>
            <id>ucsd.artifactory</id>
            <name>lib-artifacts-releases</name>
            <url>https://lib-artifacts.ucsd.edu/artifactory/maven-release-local</url>
            <snapshots>
                <enabled>false</enabled>
            </snapshots>
        </repository>

Are you able to compile without lib-artifacts?

Thanks,
Ron

**Comment by lsitu** [comment]
@rstanonik Yes. I am able to compile it with the following command:
```
mvn clean install -Ddependency-check.skip=true
```
I just ran into a dependency-check error that failed to download a json file so I disable dependency-check to build it.
cc: @thorucsd

**Comment by lsitu** [comment]
@rstanonik @thorucsd Do you still have the `lib-artifacts` issue with ACE building or not? Note that we need to review and merge PR https://github.com/Chronopolis-Digital-Preservation/audit-control-environment/pull/66 for chrondev. Thanks.

**Comment by thorucsd** [comment]
Nope.  We can bring lib-artifacts online whenever we need to.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by lsitu** [comment]
@thorucsd @rstanonik If we don't want to have lib-artifacts online, maybe we can disabled it in
https://github.com/Chronopolis-Digital-Preservation/audit-control-environment/blob/trunk/pom.xml#L71-L78 with the following code to see whether it works?
```
        <repository>
            <id>ucsd.artifactory</id>
            <name>lib-artifacts-releases</name>
            <url>https://lib-artifacts.ucsd.edu/artifactory/maven-release-local</url>
            <snapshots>
                <enabled>false</enabled>
            </snapshots>
            <releases>
                <enabled>false</enabled>
            </releases>
        </repository>
```

**Comment by rstanonik** [comment]
This has been deployed to UMIACS and TDL
